.gitignore

```diff
@@ -58,6 +58,9 @@ node_modules/
 # Coverage directory used by tools like istanbul
 coverage/
 
+#package-lock.json
```

Comment:
Can we leave package-lock.json as a file we check into git? See: https://stackoverflow.com/a/44210813

We're not changing any dependencies in this PR anyways =)

Reply:
@akrolsmir , I didn't find any package-lock.json in the code, hence thought it is safer to add to .gitignore

Reply:
My bad. removed the file.